.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [16.x, 14.x, 12.x]
+        node: [18.x, 16.x, 14.x, 12.x]
         include:
         - os: windows-latest
           node: "16.x"
```

Comment:
Can you also update this and line 18 to use Node 18 for the version-specific jobs now?

Reply:
done